backend/api_calls.py

```diff
@@ -261,17 +261,21 @@ def AddGroup(request, firebase):
   """
   results = []
   valid_args = ['!groupId', 'gameId', 'ownerPlayerId', 'allegianceFilter']
```

Comment:
blah, that would just require more special casing to handle the !groupId. Feel free to ignore.